content/en/integrations/syslog_ng.md

```diff
@@ -85,35 +91,7 @@ Configure Syslog-ng to gather logs from your host, containers, & services.
     log { source(s_src); source(s_files); destination(d_datadog); };
     ```
```

Comment:
If we're going to remove the bit about TLS encryption we should probably include a link to the `d_http` syslog-ng module to give customers some agency over making changes to the config if desired

Something like:

> More information about http destination parameters and possibilities are available in the [syslog-ng Open Source Edition Administration Guide](https://www.syslog-ng.com/technical-documents/doc/syslog-ng-open-source-edition/3.26/administration-guide/38).

Reply:
yeah we can incorporate that.